test/test_quantumchannel.jl

```diff
@@ -0,0 +1,34 @@
+using QuantumSavory
+using ResumableFunctions
+using ConcurrentSim
+using Test
+
+bell = (Z1⊗Z1 + Z2⊗Z2)/sqrt(2.0)
+regA = Register(2)
+regB = Register(2)
+initialize!((regA[1], regB[2]), bell)
+
+sim = Simulation()
+
+# Delay queue for quantum channel 
+queue = DelayQueue{RegRef}(sim, 10.0)
+
+qc = QuantumChannel(queue)
```

Comment:
there should be a constructor that does not require you to provide an explicit `DelayQueue`

Reply:
Do you mean other than the current constructor or should we modify the current constructor?

Reply:
a second constructor